test/_init_tests.js

```diff
@@ -33,6 +33,9 @@ import * as describes from '../testing/describes';
 // All exposed describes.
 global.describes = describes;
 
+// Increase the before/after each timeout since certain times they have timedout
+// during the normal 2000 allowance.
+const BEFORE_AFTER_TIMEOUT = 5000;
```

Comment:
@mkhatib It is too late for the test to change this timeout.
@jridgewell no these are hook-level timouts, only impact the before/after ( https://mochajs.org/#timeouts look at the Hook-level section)

With this PR the master build should become green again after 2 weeks. I have ran and tuned these on SauceLabs multiple times until I finally got consistent green builds. Hopefully that would last...